sdks/python/apache_beam/coders/coders_test_common.py

```diff
@@ -143,9 +143,9 @@ def test_bytes_coder(self):
 
   def test_varint_coder(self):
     # Small ints.
-    self.check_coder(coders.VarIntCoder(), *range(-10, 10))
```

Comment:
So I don't think this change is actually necessary, the unpacking argument lists operator works just fine on ranges in Python 3.